.factorypath

```diff
@@ -0,0 +1,153 @@
+<factorypath>
```

Comment:
Ugh, this thing.  It's a maven artefact.  I was doing a lot of branch-juggling with this work because of the problems I was having with tests crashing-I must have missed it.  I'll remove.